### DISCUSSION TITLE
Unable to find metadata of OrchardCore.Rules.1.4.0

### DISCUSSION THREAD

**Discussion by Gsonovb** [OrchardCMS/OrchardCore#12455] [Q&A]
I found the cause of the error
It is a Nuget cache problem, In China, the server in Hong Kong is used, but there is no file. HTTP 404 is returned

![image](https://user-images.githubusercontent.com/3164300/191380918-c66bb2a1-b06a-4236-8c39-470689d1a692.png)

![image](https://user-images.githubusercontent.com/3164300/191381061-184c65e0-76af-4fc1-87f7-4b4f453d45ad.png)

-----
Hi

I use OrchardCore ProjectTemplates template creates a CMS project and compiles it.

Received an error message

```
D:\Sources\Practices\dotnets\orchardcms\occms\occms.csproj : error NU1101: 找不到包 OrchardCore.Rules。源 C:\Program Files\do
tnet\library-packs, Microsoft Azure Service Fabric SDK, Microsoft Visual Studio Offline Packages, nuget azure cn, nuget
.cnblogs.com, nuget.org 中不存在具有此 ID 的包
```
![image](https://user-images.githubusercontent.com/3164300/191178947-c7fa6c75-a2f0-4a22-a8ad-613cb2431f3a.png)

**Comment by hishamco** [comment]
https://github.com/OrchardCMS/OrchardCore/blob/4f907ae19da5577396af7fe034cd0d266301952c/src/OrchardCore/OrchardCore.Application.Cms.Core.Targets/OrchardCore.Application.Cms.Core.Targets.csproj#L97

`OC.Rules` is there, please steps to reproduce

**Comment by Gsonovb** [comment]
Hi, @hishamco 
I started from the release version, and the steps are as follows

1. dotnet new occms --orchard-version 1.4.0 -o occms
2. cd occms
3. dotnet build

![image](https://user-images.githubusercontent.com/3164300/191366376-faaa68e1-2cbe-41c3-9748-b3a0b1112b22.png)

**Comment by hishamco** [comment]
I just finish the test, everything seems fine at my side

![OC Template](https://user-images.githubusercontent.com/3237266/191420622-fde2446e-13ca-4fb3-82dd-ebacb05c16de.png)

**Comment by hishamco** [comment]
Still I'm not sure why this happening at your side, perhaps a caching issue

/cc @sebastienros @jtkech

**Reply by Gsonovb** [reply]
Should be Nuget cache problem,
In the Chinese region download will redirect the Hong Kong server (wad-hk1-007.cloudapp.net).

**Comment by hyzx86** [comment]
要想还原这个包得挂代理。。一个妖孽的问题

**Reply by Gsonovb** [reply]
那是排查错误用的代理。
问题已经反馈给NUGET 官方解决了。

**Reply by hishamco** [reply]
Please English, otherwise we will got lost :)

**Reply by hyzx86** [reply]
> Please English, otherwise we will got lost :)

Don't care that ~, just some unimportant information
He has sent the question back to Nuget.org
